features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/admin-dashboard/site/blocks/theme-list/block.jag

```diff
@@ -0,0 +1,44 @@
+<%
+include("/jagg/jagg.jag");
+var site = require("/site/conf/site.json");
+jagg.block("theme-list", {
+    initializer: function (data) {
```

Comment:
change `initializer` function body to `function (data) { return undefined; }` to be more descriptive.